build.sbt

```diff
@@ -2065,9 +2070,36 @@ lazy val `std-database` = project
     Compile / packageBin / artifactPath :=
       `database-polyglot-root` / "std-database.jar",
     libraryDependencies ++= Seq(
+      "org.graalvm.truffle" % "truffle-api"             % graalVersion       % "provided",
```

Comment:
Why does database need `truffle-api`? Probably copied from somewhere, but the only part of the system that needs `truffle-api` is `runtime*`.

Reply:
I was using the type `org.graalvm.Value` in some of the methods, and so this was needed for Java compiler to resolve that type.
I think I won't need this anymore so I guess I can remove it.